go/tools/builders/compilepkg.go

```diff
@@ -522,6 +522,16 @@ func compileArchive(
 		}
 	}
 
+	sysoFiles := make([]string, 0, len(srcs.sysoSrcs))
```

Comment:
Can we merge this into the block above by appending `.o` and `.syso` to the same slice (and update the comment)?